src/controllers/url-controller.ts

```diff
@@ -0,0 +1,152 @@
+import { Request, Response } from 'express';
+import prisma from '../libs/prisma-client';
+
+import { convertToHyphenated, generateShortUrl } from '../utils/short-url-generator';
+import logger from '../logs/logger';
+
+const successMsg = 'Url successfully fetched';
+const notFoundMsg = 'No Url found!';
+
+//  CREATE A SHORTEN URL FROM A LONG URL
+export const createUrl = async (req: Request, res: Response) => {
+  const { url, customName } = req.body;
+
+  if (!url) return res.status(400).send({ message: 'Url is required!' });
```

Comment:
Move this check to a middleware. Also validate that the URL is valid. If I send an invalid URL (e.g "hello from URL"), the middleware should throw the error.

You might want to checkout [express-validator](https://express-validator.github.io/docs/) or [Joi](https://joi.dev/) to handle validations

Reply:
Done